core/templates/pages/signup-page/signup-page.component.spec.ts

```diff
@@ -151,7 +151,7 @@ describe('Signup page', function() {
         expect(mockWindow.location.href).toBe('/expected_url');
       });
 
-    it('should send correct information if cannot receive email updates',
+    it('should send correct information when cannot receive email updates',
```

Comment:
Should successfully signup when user opts to not receive email updates

Reply:
Done.